src/app.js

```diff
@@ -455,10 +455,13 @@ var run = function () {
 
   function editorOnChange () {
     var input = editor.getValue()
+    // save contents
+    editor.setCacheFileContent(input)
```

Comment:
There was a 300ms timeout in the previous one, that translates to 0.3 seconds.

We could make this proper and say, save every 5 seconds. Question is: should we save before every compilation too?